app/app.go

```diff
@@ -35,15 +35,45 @@ func (p *RancherProjectFactory) Create(c *cli.Context) (*project.Project, error)
 		Args:       c.Args(),
 	}
 
-	Populate(&context.Context, c)
+	for _, v := range c.GlobalStringSlice("file") {
+		context.ComposeFiles = append(context.ComposeFiles, strings.Split(v, string(os.PathListSeparator))...)
+	}
 
-	rancherComposeFile, err := resolveRancherCompose(context.ComposeFiles[0],
-		c.GlobalString("rancher-file"))
-	if err != nil {
-		return nil, err
+	templateFile := ""
+	if len(context.ComposeFiles) == 0 {
+		if _, err := os.Stat("compose.yml"); err == nil {
+			context.ComposeFiles = []string{"compose.yml"}
+			p, err := filepath.Abs("compose.yml")
+			if err != nil {
+				return nil, err
+			}
+			templateFile = p
```

Comment:
Are you saying that we still need a separate yml file that contains question even though we merged docker-compose and rancher-compose into compose.yml?